package.json

```diff
@@ -18,9 +18,11 @@
     "@babel/core": "^7.4.4",
     "@babel/polyfill": "^7.4.4",
     "@babel/preset-env": "^7.4.4",
+    "@babel/plugin-transform-runtime": "^7.4.4",
```

Comment:
should we lock this?

Reply:
Babel is one of the only libraries I trust not to make a breaking change on a patch.